build/NuSpecs/ProjectReunion-Nuget-Native.WinRt.props

```diff
@@ -6,20 +6,20 @@
     <_ProjectReunionFoundationPlatform Condition="'$(Platform)' != 'Win32'">$(Platform)</_ProjectReunionFoundationPlatform>
```

Comment:
I'm pretty sure these are unneeded (I know you didn't add them).  I believe the proper one to use is $(PlatformTarget).

Reply:
This is to deal with VS' penchant for using "Win32" in project configs rather than "x86"
I think you're right, PlatformTarget would do the equivalent, but best left for an all up pass over the Nuspecs (that'll impact not-native files as well)